docs/sdks/ios/methods.md

```diff
@@ -47,11 +47,36 @@ let isSubscribed = CleverPush.isSubscribed()
 Objective-C:
 
 ```objective-c
+// get available tags
 NSArray* channelTags = [CleverPush getAvailableTags];
 
-[CleverPush addSubscriptionTag:@"TAG_ID"];
+// add/remove tag with action callback
+[CleverPush addSubscriptionTag:@"TAG_ID" callback:^(NSString *tagId) {
```

Comment:
Nice, can you also leave the "old" method without callback in the docs? So the user knows both ways are possible.